client/client.go

```diff
@@ -279,6 +278,18 @@ type RegistrationRequest struct {
 	Nonce           *big.Int
 }
 
+func (r RegistrationRequest) toEstimator(ethClient ethClientGetter) (*bindings.ContractEstimator, error) {
+	return bindings.NewContractEstimator(r.RegistryAddress, bindings.RegistryABI, ethClient.Client())
+}
+
+func (r RegistrationRequest) toEstimateOps() *bindings.EstimateOpts {
+	return &bindings.EstimateOpts{
+		From:   r.Identity,
+		Method: "registerIdentity",
```

Comment:
Could estimator define these as constants and with some provided description or are we keeping this open for any method you can think of?